jerry.sh

```diff
@@ -1170,6 +1196,33 @@ read_manga_choice() {
     [ "$score_on_completion" = true ] && update_score "MANGA" "immediate"
 }
 
+create_room() {
+    local room_name=$(curl -s -X POST "https://backend.watchparty.me/createRoom" d '{"video":""}' | jq .name -r)
+    echo "$room_name" >"$data_dir/last_room_name.txt"
+    echo "$room_name"
+}
+
+add_video_to_room() {
+    # todo: don't print everything that's happening here, and
+    # specifically killing the process if it's running on port 12345 and 
+    # is websocat
+    
+    # Killing websocat if it's running (there's probably a better way to do that)
+    killall -9 websocat
+    
+    # Making it so the same connection can be resused
+    websocat -t -E tcp-l:127.0.0.1:12345  reuse-raw:"wss://backend.watchparty.me/socket.io/?clientId=6121fc7a-f04a-4e7e-914e-c611ceff13dd&password=&shard=2&EIO=4&transport=websocket" &
+    pid=$!
+
```

Comment:
add this line here:
```sh
trap cleanup EXIT INT TERM
```
and change `pid` to a more explicit variable name, so that it can be killed in the cleanup function